package.json

```diff
@@ -32,7 +32,7 @@
     "dotenv": "10.0.0",
     "eslint": "7.32.0",
     "eslint-plugin-prettier": "4.0.0",
-    "ethers": "5.4.7",
+    "ethers": "5.6.4",
```

Comment:
Is this a required upgrade?

Reply:
`5.6.4` does satisfy `^5.4.6`, but I wanted to check to see if there are special changes we needed, also fun website: https://semver.npmjs.com/